scipy/optimize/_differentialevolution.py

```diff
@@ -163,6 +164,14 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
 
         .. versionadded:: 1.2.0
 
+    constraints : {NonLinearConstraint, LinearConstraint, Bounds}
+        Constraints on the solver, over and about those applied by the `bounds`
```

Comment:
shouldn't "over and about" be "over and above" ?

Reply:
Yes